metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -177,10 +182,12 @@ object Embedded {
   private def downloadDependency(
       dep: Dependency,
       scalaVersion: String,
-      classfiers: Seq[String] = Seq.empty
+      classfiers: Seq[String] = Seq.empty,
+      resolution: ResolutionParams = ResolutionParams.create()
   ): List[Path] =
     fetchSettings(dep, scalaVersion)
       .addClassifiers(classfiers: _*)
+      .withResolutionParams(resolution)
```

Comment:
FWIW, it's possible to provide a custom coursier logger for the download progress https://github.com/scalameta/mdoc/blob/54f40f32ed946d3676adab35cd05bb27a6055401/mdoc-interfaces/src/main/scala/mdoc/interfaces/Mdoc.java#L17

Reply:
The default one used by Mdoc should just report things to System.err, which should be fine, no?

Reply:
It might be nice to somehow display the progress via LSP but looking at the coursier logger api it might not be so easy. This is fine